### PR TITLE
Go comments for API docs + changelog

### PR DESCRIPTION
**Pull request by leina05**
PR checklist:

- [x] Added changelog entry.

### CODE REVIEW

File: docs/changelogs/vNEXT.rst [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+Release 2021-2-3
+################
+
+==================================
+``go-oso`` VERSION
+==================================
+
+This week we are releasing ``go-oso`` as an alpha release!
+Download the package `here<NEED LINK>`_, and check out our `quick start<NEED LINK>`_!
+``go-oso`` supports Go versions 1.11+.
```
Comment (plotnick) [inline docs/changelogs/vNEXT.rst:15]: I believe we need go 1.12+.

File: languages/go/interfaces/interfaces.go [hunk 2]
```diff
@@ -1,10 +1,19 @@
 package interfaces
 
+/*
+Interface for values that can be compared.
+*/
 type Comparer interface {
+	// Should return `true` when values are equal; `false` when not equal.
 	Equal(other Comparer) bool
+	// Should return `true` when the comparer is less than `other`;`false` when not less than.
```
Comment (plotnick) [inline languages/go/interfaces/interfaces.go:9]:
```suggestion
	// Should return `true` when the comparer is less than `other`; `false` when not less than.
```

File: languages/go/oso.go [hunk 3]
```diff
@@ -4,12 +4,21 @@ import (
 	"reflect"
 )
 
+/*
+The central object to manage application policy state, e.g.
+the policy data, and verify requests.
```
Comment (plotnick) [inline languages/go/oso.go:9]:
```suggestion
The central object to manage policy state and verify requests.
```

File: languages/go/oso.go [hunk 4]
```diff
@@ -19,26 +28,46 @@ func NewOso() (Oso, error) {
 	}
 }
 
+/*
+Load in polar policies from a ".polar" file, checking that all inline queries
+succeed.
```
Comment (plotnick) [inline languages/go/oso.go:33]:
```suggestion
Load Polar policy from a ".polar" file, checking that all inline queries succeed.
```

File: languages/go/oso.go [hunk 5]
```diff
@@ -19,26 +28,46 @@ func NewOso() (Oso, error) {
...
+*/
 func (o Oso) LoadFile(f string) error {
 	return (*o.p).loadFile(f)
 }
 
+/*
+Load in polar policies from a string, checking that all inline queries succeed.
```
Comment (plotnick) [inline languages/go/oso.go:40]:
```suggestion
Load Polar policy from a string, checking that all inline queries succeed.
```

File: languages/go/query.go [hunk 6]
```diff
@@ -63,6 +70,10 @@ func (q *Query) GetAllResults() ([]map[string]interface{}, error) {
 	return results, nil
 }
 
+/*
+Get the next query result (). Returns a pointer to a map of result bindings,
```
Comment (plotnick) [inline languages/go/query.go:74]:
```suggestion
Get the next query result. Returns a pointer to a map of result bindings,
```

File: languages/python/oso/polar/polar.py [hunk 7]
```diff
@@ -87,8 +87,7 @@ def __del__(self):
         del self.ffi_polar
 
     def load_file(self, policy_file):
-        """Load in polar policies. By default, defers loading of knowledge base
-        until a query is made."""
+        """Load in polar policies from a ".polar" file."""
```
Comment (plotnick) [inline languages/python/oso/polar/polar.py:90]:
```suggestion
        """Load Polar policy from a ".polar" file."""
```

**Review by plotnick — CHANGES_REQUESTED** [review]
Suggested some minor tweaks to wording, but generally looks good. There seems to be a spurious library checked in, though: `languages/go/internal/ffi/native/macos/libpolar.a`.

**Comment by saolsen** [comment]
> Suggested some minor tweaks to wording, but generally looks good. There seems to be a spurious library checked in, though: `languages/go/internal/ffi/native/macos/libpolar.a`.

Need to update the .gitignore since the native directory moved.

**Review by plotnick — APPROVED** [review]

**Pull request merged**